### PR TITLE
fix(visualization): added more localizations

### PR DESCRIPTION
**Pull request by daragh-king-genesys**
added more localizations

✅ Closes: COMUI-2168

### CODE REVIEW

File: packages/genesys-spark-chart-components/package.json [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
   ],
   "scripts": {
-    "build": "npm run clean && npm run i18n && npm run stencil && npm run build-wrapper",
+    "build": "npm run clean && npm run i18n && npm run copy-genesys-spark-utils && npm run stencil && npm run build-wrapper",
     "build-i18n": "./scripts/build-i18n.js",
     "build-wrapper": "./scripts/wrap-stencil.js",
     "check-a11y": "node ./scripts/check-a11y.mjs",
     "check-readmes": "./scripts/check-readmes.sh",
     "clean": "rm -r ./dist ./build || true",
+    "copy-genesys-spark-utils": "node ./scripts/copy-genesys-spark-utils.mjs",
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-chart-components/package.json:17]: @katie-bobbe-genesys assume it is ok to duplicate this into this package for the moment too.

File: packages/genesys-spark-chart-components/src/components/beta/gux-visualization/tests/d3-rs-intl.spec.ts [hunk 2]
```diff
@@ -0,0 +1,2364 @@
+import { time } from '@redsift/d3-rs-intl';
+
+describe('@redsift/d3-rs-intl', () => {
+  describe('time snapshot tests', () => {
```
Comment (daragh-king-genesys) [inline packages/genesys-spark-chart-components/src/components/beta/gux-visualization/tests/d3-rs-intl.spec.ts:4]: This test is pretty pointless but it gives us something to reference if anyone comes to us with bugs in the localisations

**Comment by 321gillian** [comment]
Jenkins build didn't succeed due to lack of memory
Screenshot 2024-10-02 at 14 39 18

**Review by katie-bobbe-genesys — APPROVED** [review]
I reran the Jenkins build and it passed looks good to me

**Pull request merged**